platform-sdk/swirlds-common/src/main/java/com/swirlds/common/system/SwirldState.java

```diff
@@ -16,26 +16,15 @@
 
 package com.swirlds.common.system;
 
-import com.swirlds.common.Releasable;
 import com.swirlds.common.merkle.MerkleNode;
-import com.swirlds.common.system.events.ConsensusEvent;
+import com.swirlds.common.system.events.Event;
+import com.swirlds.common.system.transaction.Transaction;
 
 /**
  * A Swirld app is defined by creating two classes, one implementing {@link SwirldMain}, and the other
- * {@link SwirldState}, such that: <br>
```

Comment:
Why are these comments deleted?

Reply:
IMO, existing javadocs were over-fitted to the hedera/PTT use case. Also, a lot of the advice is out of date (e.g. telling the app how to synchronize methods. Let's discuss.

Reply:
this has been reverted